Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:latest
+
+WORKDIR /app
+
+COPY package*.json .
+
+RUN npm ci
+
+COPY . .
+
+RUN npm run build
```

Comment:
could you explain how we get the ENV variables into this at runtime? i.e. when the container launches?

Reply:
Similar to the Dockerfile in the old query tool using a `.env` file.